resources/views/components/game/compare-progress.blade.php

```diff
@@ -9,10 +11,18 @@
 ])
 
 <?php
-$followedUserIds = $user->followedUsers()->select(['UserAccounts.ID', 'UserAccounts.User'])->pluck('ID');
-
 $followedUserCompletion = null;
-if (!empty($followedUserIds)) {
+
+if ($user !== null) {
+    $limitedFollowedUsers = UserRelation::query()
+        ->join('UserAccounts', 'Friends.related_user_id', '=', 'UserAccounts.ID')
+        ->where('Friends.user_id', '=', $user->id)
+        ->where('Friends.Friendship', '=', UserRelationship::Following)
+        ->select('UserAccounts.ID')
+        ->orderBy('LastLogin', 'DESC')
```

Comment:
`orderBy` is no longer necessary without the limit clause all IDs will be returned.